README.md

```diff
@@ -318,6 +321,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Emby](https://github.com/MediaBrowser/MediaBrowser) - A home media server built using other popular open source technologies. - `GPLv2`
   * [Flumotion](http://flumotion.net/) - a streaming media server with intuitive graphical administration tools ([Source code](https://github.com/Flumotion/flumotion)) `LGPLv2` `Python`
   * [HTPC Manager](http://htpc.io/) - A fully responsive interface to manage all your favorite media on your HTPC. - `MIT`
+  * [Podcatcher.sh](https://github.com/DrAzraelTod/podcatcher.sh) - gets Podcasts by wget, builds m3u-Playlist, contains m3u.js that builds html5-Player for Playlists - `GPL` `Bash` `JS`
```

Comment:
I don't know if this is relevant to this list. These are mostly shell scripts to build an html index/m3u playlist for podcasts.

> Does it work?
> Well.. some of it does

I see this is your own project, we currently have no quality requirements, but I'd prefer if you let it mature a bit/improve it before adding it to the list (not invoking subshells with backticks for example)

Is this ok?